### ISSUE TITLE
Auto updates: Plugin didn't get activated  after auto-update

### ISSUE THREAD

**Issue by brbrr** [Automattic/jetpack#9262]
#### Steps to reproduce the issue
1. Grab Jetpack 5.9 from [here](https://github.com/Automattic/jetpack/archive/5.9.zip)
2. Create new JN site without Jetpack: https://jurassic.ninja/create/?shortlived&nojetpack
3. Install Jetpack 5.9 and connect
4. From Jetpack dashboard click on a link in "Plugin Updates" section
5. After redirect to Calypso - enable auto-updates from Jetpack.
6. Once plugin get updated - go back to your site, and check that Jetpack isn't active anymore

#### What I expected

Jetpack automatically activated after auto-updated

#### What happened instead

Jetpack wasn't activated after auto-updated

**Comment by oskosk** [comment]
Couldn't reproduce. Jetpack 6.0 was still active for me while after following the steps you mentioned

**Comment by brbrr** [comment]
I've reproduced it even without connecting Jetpack, so it isn't related to Auto updates.

I've updated the plugin on Plugins page, and after page refresh I got this error:
`The plugin jetpack-5.9/jetpack.php has been deactivated due to an error: Plugin file does not exist.`

![screencast 2018-04-10 12-57-25](https://user-images.githubusercontent.com/5654161/38550419-2b1a613c-3cbf-11e8-87df-6b050cecdba9.gif)

**Comment by oskosk** [comment]
Actually, I tried with getting Jetpack 5.9 from the Plugin Directory (here https://wordpress.org/plugins/jetpack/advanced/). And it worked great. 

It seems that getting the 5.9 release from the zip you mentioned, ends up installing jetpack under a directory named `jetpack-5.9`. 

The different directory name is what results in the plugin there being deactivated in favor of the updated one that will live in the `plugins/jetpack` directory.

In this case, I get this warning when reaching the plugins page:

![image](https://user-images.githubusercontent.com/746152/38550620-b2f4b8b4-3cbf-11e8-8e58-e0a3e8086f07.png)

Closing as it's not an issue we're concerned about given that we rely on Jetpack being officialy distributed through the Plugin Directory bundles.
